### ISSUE TITLE
Scope: Editorial updates (Shadi)

### ISSUE THREAD

**Issue by moekraft** [w3c/wcag-act#246]
[Accessibility Conformance Testing.docx](https://github.com/w3c/wcag-act/files/2293652/Accessibility.Conformance.Testing.docx)
The ACT Rules Format defined in this specification is used to describe rules, called ACT Rules. It is focused on ACT Rules for testing content created using web technologies, such as Hypertext Markup Language [HTML], Cascading Style Sheets [CSS2], Accessible Rich Internet Applications [WAI-ARIA], Scaleable Vector Graphics [SVG2] and more, including digital publishing. The ACT Rules Format, however, is designed to be technology agnostic, meaning that it can conceivably be used to describe test rules for other technologies.
The ACT Rules Format is also focused on ACT Rules for testing accessibility requirements defined in the Web Content Accessibility Guidelines [WCAG]. Other accessibility requirements applicable to web technologies can also be tested using ACT Rules. For example, ACT Rules could be developed to test the conformance of user agents to the User Agent Accessibility Guidelines [UAAG20]. However, the ACT Rules Format may not always be suitable to describe test rules for other types of accessibility requirements.
Because ACT Rules rarely test the entirety of an accessibility requirement, passing ACT rules does not necessarily mean that an accessibility requirement is met. It is important to understand that ACT Rules test non-conformance to accessibility requirements. In some cases conformance can be inferred from the absence of failures. Unlike WCAG sufficient techniques, ACT Rules should not be used for conformance claims unless the rule explicitly states it can be used that way. See Rule Aggregation for details.

**Comment by maryjom** [comment]
@moekraft Is there a pull request yet, or has this already been committed with a pull request? Can we close this now?  If a PR has been done, please add a reference in here.

**Comment by WilcoFiers** [comment]
PR merged, closing this issue. Due to time constraints I'm merging all of the editorials changes for a single vote.
